auto_nag/scripts/triage_owner_rotations.py

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from typing import List
+
+from libmozdata.bugzilla import BugzillaComponent
+from requests import HTTPError
+
+from auto_nag import logger
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.component_triagers import ComponentTriagers, TriageOwner
+
+
+class TriageOwnerRotations(BzCleaner):
+    def __init__(self) -> None:
+        super().__init__()
+        self.query_url = None
+        self.has_put_errors = False
+
+    def description(self) -> str:
+        return "Triage owners that got updated"
+
+    def get_extra_for_template(self):
+        return {"has_put_errors": self.has_put_errors}
+
+    def update_triage_owners(self, new_triage_owners: List[TriageOwner]) -> set:
+        failures = set()
+        for new_triager in new_triage_owners:
+            change = {"triage_owner": new_triager.bugzilla_email}
+            if self.dryrun or self.test_mode:
+                logger.info(
+                    "The component %s::%s will be updated with:\n%s",
+                    new_triager.product,
+                    new_triager.component,
+                    change,
+                )
+            else:
+                try:
+                    BugzillaComponent(
+                        new_triager.product,
+                        new_triager.component,
+                    ).put(change)
+                except HTTPError as err:
```

Comment:
Maybe we should implement a retry in `libmozdata` for `BugzillaComponent`

Reply:
Or use a library like `tenacity` here, so we decouple the retrying from the implementation of the API calls.

Reply:
I like this option more (implement the retry here, not in the API). I initially suggested the retry in `libmozdata` for consistency:

https://github.com/mozilla/libmozdata/blob/c092351450dae908c4f7184bebb492d304910035/libmozdata/bugzilla.py#L101

Reply:
Done in 7913ab960c9dacb7310d7781428a3e45fa871006